docs/docs/webui.md

```diff
@@ -19,7 +20,7 @@ There are also two included example YAML files that make use of this configurati
 * [config/local_scheduler.yml](/mesos/chronos/blob/master/config/local_scheduler.yml#L4)
 * [config/local_scheduler_nozk.yml](/mesos/chronos/blob/master/config/local_scheduler_nozk.yml#L7)
 
-## Build Requirements
+### Build Requirements
 
```

Comment:
Makes sense, we can add a disclaimer saying, "When building and optimizing the assets, make sure you have at least 1GB of RAM available" without linking to the issue.